changelog.d/9098.bugfix

```diff
@@ -0,0 +1 @@
+Fix reactor being passsed as ip_whitelist to BlacklistingAgentWrapper from MatrixFederationAgent.
```

Comment:
I think we should rename this to `misc` since there's no outward facing bug that comes from this.

```suggestion
Fix the wrong arguments being passed to `BlacklistingAgentWrapper` from `MatrixFederationAgent`.
```

You can add `Contributed by <your name or handle>.` if you'd like!

Reply:
Updated the changelog and renamed to `9098.misc`!